x-pack/plugins/lists/common/utils.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { NamespaceType } from './schemas/types';
+import { NamespaceTypeArray } from './schemas/types/default_namespace_array';
+import {
+  SavedObjectType,
+  exceptionListAgnosticSavedObjectType,
+  exceptionListSavedObjectType,
+} from './types';
+
+export const getSavedObjectType = ({
```

Comment:
I know you're still cleaning up this PR, but it seems like we need to update references to `getSavedObjectType` throughout the codebase. Currently it's not possible to create a new rule because the `create_rules_route` relies on `createEndpointList()`, which in turn relies on `getSavedSavedObjectType()`. But the import statement is out of date in the `create_endpoint_list` file.
![image](https://user-images.githubusercontent.com/616158/102220877-6636d300-3eaf-11eb-81fb-6af7e3570f0a.png)

Reply:
Ah, sorry. I thought I had pushed my last commit last night, but looks like I hadn't. This is addressed there!

Reply:
Pushed the code in, sorry about that!